roboflow/models/instance_segmentation.py

```diff
@@ -0,0 +1,72 @@
+import io
+import urllib
+
+import requests
+from PIL import Image
+from requests_toolbelt.multipart.encoder import MultipartEncoder
+
+from roboflow.config import INSTANCE_SEGMENTATION_MODEL, INSTANCE_SEGMENTATION_URL
+from roboflow.util.image_utils import validate_image_path
+from roboflow.util.prediction import PredictionGroup
+
+
+class InstanceSegmentationModel:
+    def __init__(
+        self,
+        api_key,
+        version_id,
+    ):
+        """
+        :param api_key: Your API key (obtained via your workspace API settings page)
+        :param version_id: The ID of the dataset version to use for predicting
+        """
+        self.__api_key = api_key
+        self.id = version_id
+
+        version_info = self.id.rsplit("/")
+        dataset_id = version_info[1]
+        version = version_info[2]
+
+        self.api_url = f"{INSTANCE_SEGMENTATION_URL}/{dataset_id}/{version}"
+
+    def predict(self, image_path):
```

Comment:
Let's add confidence as a param as well, should flow through like the object detection confidence param does